pkg/app/ops/firestoreindexensurer/indexes.json

```diff
@@ -1,209 +1,306 @@
-        }
-      ]
-    },
-    {
-      "collectionGroup": "Command",
-      "queryScope": "COLLECTION",
-      "fields": [
-        {
-          "fieldPath": "Status",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "CreatedAt",
-          "order": "ASCENDING"
-        }
-      ]
-    },
-    {
-      "collectionGroup": "Deployment",
-      "queryScope": "COLLECTION",
-      "fields": [
-        {
-          "fieldPath": "ApplicationId",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "UpdatedAt",
-          "order": "DESCENDING"
-        }
-      ]
-    },
-    {
-      "collectionGroup": "Deployment",
-      "queryScope": "COLLECTION",
-      "fields": [
-        {
-          "fieldPath": "EnvId",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "UpdatedAt",
-          "order": "DESCENDING"
-        }
-      ]
-    },
-    {
-      "collectionGroup": "Deployment",
-      "queryScope": "COLLECTION",
-      "fields": [
-        {
-          "fieldPath": "Kind",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "UpdatedAt",
-          "order": "DESCENDING"
-        }
-      ]
-    },
-    {
-      "collectionGroup": "Deployment",
-      "queryScope": "COLLECTION",
-      "fields": [
-        {
-          "fieldPath": "ProjectId",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "UpdatedAt",
-          "order": "DESCENDING"
-        }
-      ]
-    },
-    {
-      "collectionGroup": "Deployment",
-      "queryScope": "COLLECTION",
-      "fields": [
-        {
-          "fieldPath": "Status",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "UpdatedAt",
-          "order": "DESCENDING"
-        }
-      ]
-    },
-    {
-      "collectionGroup": "Event",
-      "queryScope": "COLLECTION",
-      "fields": [
-        {
-          "fieldPath": "EventKey",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "Name",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "ProjectId",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "CreatedAt",
-          "order": "DESCENDING"
-        }
-      ]
-    },
-    {
-      "collectionGroup": "Event",
-      "queryScope": "COLLECTION",
-      "fields": [
-        {
-          "fieldPath": "ProjectId",
-          "order": "ASCENDING"
-        },
-        {
-          "fieldPath": "CreatedAt",
-          "order": "ASCENDING"
-        }
-      ]
-    }
-  ],
-  "fieldOverrides": []
-}
+[
+  {
+    "collectionGroup": "Application",
+    "queryScope": "COLLECTION",
+    "fields": [
+      {
+        "fieldPath": "Disabled",
+        "order": "ASCENDING",
+        "arrayConfig": ""
+      },
+      {
+        "fieldPath": "UpdatedAt",
+        "order": "DESCENDING",
+        "arrayConfig": ""
+      },
+      {
+        "fieldPath": "__name__",
```

Comment:
Do you know why was this included?

Reply:
Actually, this `__name__` field is automatically added for all indexes when it wasn't explicitly given.

Reply:
Yes, I know. Just want to know why they were created.
👍

Reply:
Ah, okay. I don't know why it does exactly. I was looking for it but the doc says just "it's required".